docs/README.md

```diff
@@ -87,5 +133,17 @@ With this `kubectl` alias I can use commands like
 | `kubectl get pod --all-namespaces` | `k g p -a` |
 | `kubectl apply -R -f somefolder` | `k a -f somefolder` |
 | ``kubectl config set-context `kubectl config current-context` --namespace production`` | `k ns production` |
+| ... | and so on... |
+
+##### Normal aliases or subshell executions are also possible
 
+```
+random-password = sh -c "dd if=/dev/urandom bs=1 count=12 2>/dev/null | base64 -w 0 | rev | cut -b 2- | rev";
```

Comment:
Right. You can also use this technique for more complex command using the `!` char at anylevel
```
myutils = echo "MyUtils" {
random-password = !sh -c "dd if=/dev/urandom bs=1 count=12 2>/dev/null | base64 -w 0 | rev | cut -b 2- | rev";
}
```

and then run `myutils random-password`